src/EnergyPlus/StandardRatings.cc

```diff
@@ -464,248 +465,260 @@ namespace StandardRatings {
 
         CheckCurveLimitsForIPLV(state, ChillerName, ChillerType, CondenserType, CapFTempCurveIndex, EIRFTempCurveIndex);
 
-        // IPLV calculations:
-        for (RedCapNum = 0; RedCapNum < NumOfReducedCap; ++RedCapNum) {
-            if (CondenserType == DataPlant::CondenserType::WaterCooled) {
-                // get the entering water temperature for the reduced capacity test conditions
-                if (ReducedPLR[RedCapNum] > 0.50) {
-                    EnteringWaterTempReduced = 8.0 + 22.0 * ReducedPLR[RedCapNum];
-                } else {
-                    EnteringWaterTempReduced = 19.0;
-                }
-                CondenserInletTemp = EnteringWaterTempReduced;
-            } else if (CondenserType == DataPlant::CondenserType::AirCooled) {
-                // get the outdoor air dry bulb temperature for the reduced capacity test conditions
-                if (ReducedPLR[RedCapNum] > 0.3125) {
-                    EnteringAirDryBulbTempReduced = 3.0 + 32.0 * ReducedPLR[RedCapNum];
-                } else {
-                    EnteringAirDryBulbTempReduced = 13.0;
-                }
-                CondenserInletTemp = EnteringAirDryBulbTempReduced;
-            } else { // EvaporativelyCooled Condenser
-                // get the outdoor air wet bulb temperature for the reduced capacity test conditions
-                EnteringAirWetBulbTempReduced = 10.0 + 14.0 * ReducedPLR[RedCapNum];
-                CondenserInletTemp = EnteringAirWetBulbTempReduced;
+        Real64 IPLV = 0.0;                        // current IPLV
+        Real64 EvapOutletTemp = 0.0;              // evaporator leaving water temperature, deg C
+        Real64 constexpr EvapOutletTempSI = 7.0;  // evaporator LWT, 2011 AHRI Std 551 / 591(SI)
+        Real64 constexpr EvapOutletTempIP = 6.67; // evaporator LWT, 2011 AHRI Std 550 / 590(IP), (44F)
+        bool ReportStdRatingsOnce = true;
+
+        for (const auto &AhriStd : AhriChillerStdNamesUC) {
```

Comment:
Just iterate over the enumeration, why iterate over the strings and then convert to the enumeration?

Reply:
OK. Thanks. I am going to modify it as follows:

```
        for (const auto &AhriStd : {AhriChillerStd::AHRI550_590, AhriChillerStd::AHRI551_591}) {
            if (AhriStd == AhriChillerStd::AHRI550_590) {
                EvapOutletTemp = EvapOutletTempIP;
                ReportStdRatingsOnce = true;
            } else if (AhriStd == AhriChillerStd::AHRI551_591) {
                EvapOutletTemp = EvapOutletTempSI;
                ReportStdRatingsOnce = false;
```